content/en/observability_pipelines/guide/custom-metrics-governance.md

```diff
@@ -0,0 +1,215 @@
+
+## Drop metrics missing specific tags or the metric's namespace
+
+### Problem
+
+It is useful to break down your overall metrics usage with the [Usage Attribution][6] feature, by adding tags like `team` or `application`. This lets you see how many custom metrics a particular team or application is generating. However, custom metrics might be submitted without those crucial tags. This makes it difficult to attribute the metrics back to the specific team, service, or application generating them. 
+
+### Solution
+
+To prevent that issue, use Observability Pipelines to drop metrics missing tags that are useful for Usage Attribution. This can be done before the metrics are added to your account's custom metrics usage. 
+
+Observability Pipelines has a wide array of functions that can transform your metrics data before it is sent to Datadog. For example, use the `filter` transform to drop metrics that are missing specific tag keys. The following component filters out any metrics that do not have a `team_tag`, ensuring those metrics are dropped in your Observability Pipelines configuration.
+
+```yaml
+transforms:
+  my_transform_id:
+    type: filter
+    inputs:
+      - my-source-or-transform-id
+    condition: |
+      exists(.tags.<team_tag>)
+```
+
+Similarly, in cases where you want to filter out metrics by namespace instead of tags, use the following configuration.
+
+```yaml
+transforms:
+  my_transform_id:
+    type: filter
+    inputs:
+      - my-source-or-transform-id
+    condition: |2
+       .namespace == "foo" 
+```
+## Remove custom metric tags
+
+### Problem
+
+Metrics tags can provide better visibility into specific hosts, pods, applications, and services. However, some metrics may have tags that are rarely queried or useful for your visualizations or investigative workflows. While you can use Metrics without Limits™ to exclude potential superfluous tags automatically (after a metric has been ingested), you cannot drop tags on metrics (prior to metric ingestion) to prevent these tags from contributing towards your organization's ingested metrics usage. To address this issue, you can use Observability Pipelines to do one of the following **before the metric is ingested**:
+
+- [Drop one tag](#solution-1-drop-one-tag)
+- [Define an allowlist array of tags to keep](#solution-2-define-an-allowlist-array-of-tags-to-keep)
+- [Define a blocklist array of tags to drop](#solution-3-define-a-blocklist-array-of-tags-to-drop)
+- [Define an allowlist of valid tags in a Reference Table](#solution-4-define-an-allowlist-of-valid-tags-in-a-reference-table)
+
+**Note**: Removing tags on metrics prior to ingestion impacts the mathematical accuracy of your metrics' queries. Using Metrics without Limits™ allows you to define important tags at any time, without impacting the mathematical accuracy of your metrics' queries.
```

Comment:
Can we move this Note a bit earlier to be right before the sentence:  "To address this issue, you can use Observability Pipelines to do one of the following **before the metric is ingested**:"